clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -386,17 +400,19 @@ object GarbageCollector {
     if (!storageNSForSdkClient.endsWith("/")) {
       storageNSForSdkClient += "/"
     }
+    
+    var removed : DataFrame = spark.emptyDataFrame
 
-    val removed =
-      remove(storageNSForSdkClient,
+    if (!hc.getBoolean(LAKEFS_CONF_DEBUG_GC_NO_DELETE_KEY, false)) {
```

Comment:
This line contains three negatives: `! ... NO_DELETE ... false`!  Maybe have the key name in positive form with default "true"?

Reply:
Don't want to change the configuration name, which I think is good, for the sake of simplifying the code.
However, after implementing your ternary suggestion, we are now left with a double negative.